Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish.Tests/QuorumQueueTests.cs

```diff
@@ -0,0 +1,350 @@
+        Assert.AreEqual("test message", readValues.Message);
+        Assert.AreEqual("String", result.DataFormat);
+        Assert.AreEqual("test message", result.DataString);
+        Assert.IsTrue(result.DataByteArray.SequenceEqual(Encoding.UTF8.GetBytes("test message")));
+        Assert.AreEqual(0, result.Headers.Count);
+    }
+
+    [TestMethod]
+    public void TestPublishAsByteArray_WithoutHeaders()
+    {
+        Connection connection = new()
+        {
+            Host = TestHost,
+            Username = "agent",
+            Password = "agent123",
+            RoutingKey = "quorum",
+            QueueName = "quorum",
+            Create = false,
+            Durable = false,
+            AutoDelete = false,
+            AuthenticationMethod = AuthenticationMethod.Host,
+            ExchangeName = ""
+        };
+
+        Input input = new()
+        {
+            DataByteArray = Encoding.UTF8.GetBytes("test message"),
+            InputType = InputType.ByteArray,
+            Headers = null
+        };
+
+        var readValues = new Helper.ReadValues();
+        var result = RabbitMQ.Publish(input, connection);
+        Helper.ReadMessage(readValues, connection);
+        Assert.IsNotNull(readValues.Message);
+        Assert.AreEqual("test message", readValues.Message);
+        Assert.AreEqual("ByteArray", result.DataFormat);
+        Assert.AreEqual("test message", result.DataString);
+        Assert.IsTrue(result.DataByteArray.SequenceEqual(Encoding.UTF8.GetBytes("test message")));
+    }
+
+    [TestMethod]
+    public void TestURIConnection()
+    {
+        Connection connection = new()
+        {
+            Host = TestUri,
+            RoutingKey = "quorum",
+            QueueName = "quorum",
+            Create = false,
+            Durable = false,
+            AutoDelete = false,
+            AuthenticationMethod = AuthenticationMethod.URI,
+        };
+
+        Input input = new()
+        {
+            DataString = "test message",
+            InputType = InputType.String,
+
+            Headers = new Header[] {
+            new Header { Name = "X-AppId", Value = "application id" },
+            new Header { Name = "X-ClusterId", Value = "cluster id" },
+            new Header { Name = "Content-Type", Value = "content type" },
+            new Header { Name = "Content-Encoding", Value = "content encoding" },
+            new Header { Name = "X-CorrelationId", Value = "correlation id" },
+            new Header { Name = "X-Expiration", Value = "100" },
+            new Header { Name = "X-MessageId", Value = "message id" },
+            new Header { Name = "Custom-Header", Value = "custom header" }
+        }
+        };
+
+        var readValues = new Helper.ReadValues();
+        var result = RabbitMQ.Publish(input, connection);
+        Helper.ReadMessage(readValues, connection);
+
+        Assert.IsTrue(!string.IsNullOrEmpty(readValues.Message) && readValues.Message.Equals("test message"));

```

Comment:
Please move these checks to their own asserts and use `AreEqual` for equal comparisons.